Sources/Mantis/CropViewController/CropViewController.swift

```diff
@@ -482,3 +609,67 @@ extension CropViewController {
         cropView.getExpectedCropImageSize()
     }
 }
+
+extension CropViewController: TransformDelegate {
+   
+    public func enableUndo(_ enable: Bool) {
```

Comment:
Can we remove `public` for delegate functions here?

Reply:
changes made.